### PR TITLE
Fix type enclosing on methods

### PR DESCRIPTION
**Pull request by rgrinberg**
What it says on the tin

### CODE REVIEW

File: src/analysis/type_enclosing.ml [hunk 1]
```diff
@@ -33,6 +33,26 @@ let from_nodes ~path =
     | Module_declaration_name {md_type = {mty_type = m}}
     | Module_type_declaration_name {mtd_type = Some {mty_type = m}} ->
       ret (Modtype (env, m))
+    | Class_field
+        { cf_desc =
+            Tcf_method (_, _, Tcfk_concrete (_, {exp_type = { desc = Tarrow (_, _, t, _) } }))
```
Comment (voodoos) [inline src/analysis/type_enclosing.ml:38]: Can you break this long line ?

**Review by voodoos — APPROVED** [review]
Looks good to me, thank you !
Fixes #1124

Can you break the long line and add a Changelog entry ?

File: src/analysis/type_enclosing.ml (hunk 1 above)
Comment (rgrinberg) [reply src/analysis/type_enclosing.ml]: I did. Feel free to adjust further as the formatting is still a little weird.

**Comment by rgrinberg** [comment]
@voodoos done!

**Pull request merged**